plasmapy/plasma/spheromak_solutions.py

```diff
@@ -0,0 +1,113 @@
+import math
+import numpy as np
+import scipy.special
+import sympy.functions.special.bessel
+
+from sympy import Derivative
+
+
+class solution:
```

Comment:
```suggestion
class ForceFreeSpheromak:
```
...since I'm thinking we want to emphasize that it's force-free.